src/com/egs/arsen/task1/Main.java

```diff
@@ -0,0 +1,110 @@
+package com.egs.arsen.task1;
+
+import java.io.*;
+import java.util.Scanner;
+
+public class Main
+{
+
+    public static void main(String[] args) throws Exception {
+        Scanner scanner = new Scanner(System.in);
+        boolean state = true;
+
+
+        File file = new File("wordFile.txt");
+        File fileResult = new File("ResultFile.txt");
+
+        try {
+            if (file.exists()) {
+                file.delete();
+
+
+            }
+            if (!file.exists()) {
+
+                fileResult.createNewFile();
+
+            }
+            boolean exists = file.exists();
+            if (exists) {
+                System.out.println(" Write your text and pleas put , between words");
+            }
+
+
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        while (state) {
+            String str = scanner.nextLine();
+            if (str.equalsIgnoreCase("exit")) {
+
+
+                state = false;
+            } else {
+                try {
+
+                    String s[] = str.split(",");
+                    writeToFile(file, str);
+
+
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            }
+
+        }
+        modify(file, fileResult);
+
+
+    }
+
+    public static void writeToFile(File file, String data) throws IOException {
+        BufferedWriter writer = new BufferedWriter(new FileWriter(file, true));
```

Comment:
flush not necessary to call
close should be inside finally block
or use try with resources

Reply:
if i use only flush without  close , it`s possible ? i mean that flush already has close method inside, or its not right?

Reply:
if you call close, flush will be called internally